packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -203,10 +187,10 @@ const configureRequest = async (
 const parseDataFromResponse = (response) => {
   const dataBuffer = Buffer.from(response.data);
   // Parse the charset from content type: https://stackoverflow.com/a/33192813
-  const charset = /charset=([^()<>@,;:\"/[\]?.=\s]*)/i.exec(response.headers['Content-Type'] || '');
+  const charset = /charset=([^()<>@,;:"/[\]?.=\s]*)/i.exec(response.headers['Content-Type'] || '');
```

Comment:
@mirkogolze Is the removal `\` intentional ?

Reply:
Inside brackets `[...]` you don't need to escape `"`
![image](https://github.com/usebruno/bruno/assets/66121941/38aa89fd-fdb8-4602-9dc2-4e7100479739)
![image](https://github.com/usebruno/bruno/assets/66121941/41b3dc25-fbdd-47d6-b0d8-0164ab4b3495)

And my IntelliJ really helps me here.